README.md

```diff
@@ -52,10 +52,60 @@ If you prefer the programming ligature so far, I recommend Fira Code because it
 
 ## Installation
 
+### macOS
+
+1. Download font files
+1. Unzip the archive and install the font:
```

Comment:
Hmmm, it seems very redundant to have separate sections for macOS and Windows and the exact same explanation in them. If you like, these two steps are the same for Linux. 🤔